src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -91,6 +94,7 @@ uint8_t gListUint8Data[kAttributeListLength];
 OctetStringData gListOctetStringData[kAttributeListLength];
 OctetStringData gListOperationalCert[kAttributeListLength];
 Structs::TestListStructOctet::Type listStructOctetStringData[kAttributeListLength];
+OctetStringData gStructAttributeByteSpanData;
 Structs::SimpleStruct::Type gStructAttributeValue = { 0,          false,      SimpleEnum::kValueA,
```

Comment:
Do we support `Nullable` fields in the attribute structures? Does it make sense to include the check for them in those tests?

Reply:
> Do we support Nullable fields in the attribute structures?

Not sure whether you mean a struct with nullable fields or an attribute that is struct-typed but nullable, but we support both.

> Does it make sense to include the check for them in those tests?

We have existing tests for reading/writing structs with nullable fields (a list of them).

Are you asking about a struct-typed but nullable attribute? Or something else?  @Morozov-5F

Reply:
I meant structures with nullable fields. If we already have those tests then I don't think we need to do anything else in this PR